### DISCUSSION TITLE
Maui MediaElement support for dash

### DISCUSSION THREAD

**Discussion by modigithub** [CommunityToolkit/Maui#1026] [Q&A]
hi, I read that MediaElement supports HLS.
(https://devblogs.microsoft.com/dotnet/announcing-dotnet-maui-communitytoolkit-mediaelement/)
![image](https://user-images.githubusercontent.com/108171635/220835574-14c844fe-da65-451d-b659-fa7d600f6850.png)
Does this apply to all underlying devices?

Will DASH also be supported in the future or is nothing planned here? (like dash.js in html: http://reference.dashif.org/dash.js/latest/samples/)

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
